collect-settings.rst

```diff
@@ -3,7 +3,7 @@ Collect Settings and Security
 
 Settings are accessible from the :guilabel:`⋮` menu.
 
-There are two kings of setting, **General Settings** and **Admin Settings**. As the names suggest, this is really helpful when the person who is collecting the data and the owner of the device are not same. So the admin can set an admin password and choose what all general settings the enumerator (in this case, who is not the owner) can access. This feature can also hide/show the items displayed in the main menu and the options available in the options menu while filling the form. 
+There are two kings of setting, **General Settings** and **Admin Settings**. This is used when the person who is collecting the data and the owner of the device are not same. Admin can set an *admin password* and choose what all general settings the enumerator (in this case not the owner) can access. You can also hide/show the items displayed in the main menu and the options available in the options menu while filling the form. 
```

Comment:
s/kings/kinds

Reply:
> This is used when the person who is collecting the data and the owner of the device are not same.

I don't think you mean to suggest that admin and general settings are only used when the owner is not the enumerator.

Reply:
"generally useful"? or just 'useful' is okay?

Reply:
i don't understand your question. I wouldn't say "useful" at all. Don't tell people something is "useful for x" just say how things work and what they do, and what is possible.

Also...
This information is already in the document. I'm not sure why you feel the need to add it here.